app-backend/batch/src/main/scala/com/azavea/rf/batch/util/S3.scala

```diff
@@ -109,9 +109,11 @@ object S3 {
 
   /** Set credentials in case Hadoop configuration files don't specify S3 credentials. */
   def setCredentials(conf: Configuration, credentialsProviderChain: AWSCredentialsProvider = new DefaultAWSCredentialsProviderChain): Configuration = {
-    conf.set("fs.s3.impl", classOf[org.apache.hadoop.fs.s3native.NativeS3FileSystem].getName)
-    conf.set("fs.s3.awsAccessKeyId", credentialsProviderChain.getCredentials.getAWSAccessKeyId)
-    conf.set("fs.s3.awsSecretAccessKey", credentialsProviderChain.getCredentials.getAWSSecretKey)
+    if(conf.isKeyUnset("fs.AbstractFileSystem.s3a.impl")) {
```

Comment:
Hm. According to the EMR documentation, it doesn't seem like we should be using `s3a`. I don't think that implementation has support for the IAM credential provider chain (or potentially credential provider chains at all):

> For Amazon EMR, both the s3:// and s3n:// URIs are associated with the EMR filesystem and are functionally interchangeable in the context of Amazon EMR. For consistency sake, however, it is recommended to use the s3:// URI in the context of Amazon EMR.
> 
> The s3a:// URI is not compatible with Amazon EMR. For more information, see File Systems Compatible with Amazon EMR and S3 Support in Apache Hadoop.

See: https://aws.amazon.com/premiumsupport/knowledge-center/emr-file-system-s3/